test/api/methods.js

```diff
@@ -183,13 +183,17 @@ describe("API: JSDOM class's methods", () => {
         assert.strictEqual(dom.window.top, newTop);
       });
 
-      it("should reconfigure the window.top property (tested from the inside)", () => {
+      it("should reconfigure the window.top property (tested from the inside)", { async: true }, t => {
```

Comment:
Hmm. So the change is that here we're no longer executing inline script synchronously? That seems bad. Although I know we were not perfect at it, browsers do execute inline scripts sync, so preserving that seems better if we can...

Reply:
I'm not sure if I'm talking to soon, but looks like now this is not necessary.
My theory is that the changes in `dummyPromise` for `Promise.resolve()` has helped with this.